lib/hosting/hetzner_apis.rb

```diff
@@ -2,8 +2,6 @@
 
 require "excon"
 class Hosting::HetznerApis
-  FailoverSubnet = Struct.new(:ips, :subnets, :failovers)
```

Comment:
Getting rid of this is unrelated right?

Reply:
yup, just realized it's never used, considering the size of the PR, wanted to squish that in

Reply:
I'd just shave it off into its own commit, though you can stuff it in the PR. I am generally more relaxed about PR metadata, since I really only load it when I have to do doubly-invasive root-cause analysis.